lib/node_modules/@stdlib/stats/kde2d/lib/epanechnikov.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+
+// MODULES //
+
+var abs = require( '@stdlib/math/base/special/abs' );
+var isnan = require( '@stdlib/assert/is-nan' );
+var pow = require( '@stdlib/math/base/special/pow' );
+
+
+// MAIN //
+
+/**
+* Computes the Epanechnikov kernel for a value.
+*
+* @private
+* @param {number} u - value for which we wish to calculate the Epanechnikov kernel
+* @returns {number} the value for the kernel at u
+*
+* @example
+* var u = 5;
+* out = epanechnikov(u); // returns 0
+*/
+function epanechnikov(u) {
+	var absU;
+
+	if (isnan(u)) {
+		return NaN;
+	}
+
+	absU = abs(u);
+	if (absU > 1) {
+		return 0;
+	}
+	return 0.75 * (1 - pow(u, 2));
```

Comment:
It has been our convention to always use decimal points for floating-point numbers unless they are used in integer calculations. So above should become `0.75 * (1.0 - pow(u, 2.0))`.

Reply:
Resolved for this file!

Reply:
Resolved for all files.